compiler/luci/lang/include/luci/IR/Module.h

```diff
@@ -46,6 +46,8 @@ class Module final
 public:
   void add(std::unique_ptr<loco::Graph> &&g);
 
+  void remove(size_t idx);
```

Comment:
Is it possible to make a copy of module without this `idx` subgraph and replace it?